ops/packaging/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG PHP_VERSION=7.1
-FROM php:7.2-fpm-buster
+ARG PHP_VERSION=7.2.34
+FROM php:${PHP_VERSION}-fpm-buster
 
```

Comment:
It kind of feels like the Dockerfiles in `ops/packaging` have the PHP version hardcoded. So for example if we upgrade the PHP version in `.env` to 7.3 then the value of `PHP_VERSION` in these Dockerfiles need to be updated too.

Reply:
Hi @pli888 
if it's because of the ``ARG`` clause,  its value is a default value that in normal circumanstances would never be used 
since PHP_VERSION is populated wth value either set in ``.env`` or ``.gitlab-cy.yml``.
Therefore, there is no need to change the default value  for ``ARG`` whatsoever when we upgrade the version of PHP. 
If I could, I would prefer leave ``ARG`` empty, but the Dockerfile syntax force us to put a value in there.
Maybe I should set it something like ``ARG=null`` to avoid any confusion

Unless we think that when extraordinary circumstances where neither ``.env`` nor ``.gitlab-cy.yml`` set PHP_VERSIOM, the container image should still build with some version of PHP.

Reply:
> If I could, I would prefer leave ARG empty, but the Dockerfile syntax force us to put a value in there.

Ok, from the Dockerfile reference, it seems that we can use ``ARG`` without a default values

>Unless we think that when extraordinary circumstances where neither .env nor .gitlab-cy.yml sets PHP_VERSION, the container image should still build with some version of PHP.

No, if PHP_VERSION is not set from ``.env`` or ``.gitlab-cy.yml``, we don't want the container image to build at all, as this is abnormal circumstances (that surely willl result in other issues anyways), in which we don't weant to operate

So, I've removed the default value